html/parking/index.html

```diff
@@ -1,10 +1,53 @@
-<script>
-function release(target){
-  target.parentElement.innerHTML = "One moment please"
-}
-</script>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="/common/styles.css" media="screen" rel="stylesheet" type="text/css" />
+    <title>Parking</title>
+    <script>
+      function release(target){
+        document.getElementById(target).innerHTML = "One moment please";
```

Comment:
The click event of the button needs to be disabled also to prevent bash clicking.